src/pages/workspace/distanceRates/PolicyDistanceRateTaxRateEditPage.tsx

```diff
@@ -78,16 +63,21 @@ function PolicyDistanceRateTaxRateEditPage({route, policy}: PolicyDistanceRateTa
                 shouldEnableMaxHeight
                 testID={PolicyDistanceRateTaxRateEditPage.displayName}
             >
-                <HeaderWithBackButton
-                    title={translate('workspace.taxes.taxRate')}
-                    shouldShowBackButton
-                />
-                <SelectionList
-                    sections={[{data: taxRateItems}]}
-                    ListItem={RadioListItem}
-                    onSelectRow={onTaxRateChange}
-                    initiallyFocusedOptionKey={taxRateItems.find((item) => item.isSelected)?.keyForList}
-                />
+                {({insets}) => (
+                    <>
+                        <HeaderWithBackButton
+                            title={translate('workspace.taxes.taxRate')}
+                            shouldShowBackButton
```

Comment:
Does header needs to be wrapped in the function?

Reply:
Since we used the `insets`, I believe we should do that.
We also did that in other places where we wanted to used `insets` ie `BaseSidebarScreen` or `WorkspaceNewRoomPage`
https://github.com/Expensify/App/blob/633e708dfe3d2aba47dbb2091ba1f0f8683bfaaf/src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.tsx#L40-L46